docs/_documentations/overview.md

```diff
@@ -9,21 +9,72 @@ type: document
 
-You can use codewind to develop and debug your containerized projects from within an IDE.
+From creating your project, all the way to debugging and optimising, Codewind does the heavy-lifting so that you can simply focus on rapidly iterating your code. Expose the metrics and logs that you need and perform load testing at the click of a button, and relax in a flow that will feel the same as a traditional application flow.
 
-Write code, track application and build statuses, view project logs, and run your application. Codewind supports development of Appsody, Go, Java Lagom, Microprofile/Java EE, Node.js, Python, Spring, and Swift containerized projects. In addition, the tools support easily debugging Microprofile/Java EE, Spring, and Node.js applications.
+When it comes to running your microservices on the cloud, Codewind enables you to build and run Docker-containerised applications on [Kubernetes](https://kubernetes.io/) container orchestration applications, including [Openshift](https://www.openshift.com/), without the need for in-depth knowledge of container technology.
 
+## How does Codewind help me develop containerised applications?
+Codewind improves your inner loop experience, enabling you to create a microservice quickly, rapidly iterate on changes and make improvements to performance thanks to the following features:
 
-# Features
+1. **Access to pre-configured containerised project templates** Codewind provides pre-configured, Docker-containerised project templates ADD URL covering several languages including [Node.js](https://nodejs.dev/), Java, Python and [Swift](https://swift.org/), and several frameworks such as Express, Spring Boot and Open Liberty ADD URLS, with more on the way. [Quickly create and deploy microservices in languages and frameworks that you're familiar with](https://www.youtube.com/watch?v=zKMggp10gq4&t=12s). Modify the pre-configured projects to develop your customised microservice.
+
+2. **Ability to add your own applications** Access the seamless container development and deployment experience on your IDE that Codewind offers for your current projects.
+
+3. **Auto-build-and-run on code changes** As you develop, Codewind *automatically* pushes your code changes to your container as efficiently as possible. 
+
+4. **Ability to toggle between auto-build and manual build settings** Have control over when you want to implement Codewind's auto-build feature.
+
+5. **Project details displayed in a user-friendly panel on your IDE** Use Codewind's Project Overview interface to instantly view the endpoint, container ID, location on disk, build and run status, and ports.
+
+6. **Access application, build and container logs and run commands from within the container without leaving your IDE** Codewind integrates with your IDE's terminal window, 'problems' and 'output' panels for a smooth test and debug experience. Additionally, you can open a shell session *inside* your container with a simple click.
```

Comment:
- Comma after "logs" because of Oxford comma.
- Perhaps capitalize the panel names instead of using the quotation marks? (American English also uses "double quotation marks" instead of 'single quotation marks,' which seems to be the opposite of the British standard. For example, in a novel, American English would put character dialogue in double quotation marks and then use single quotation marks for a quote within a quote.)
- Maybe use "and" after "terminal window" instead of a comma?

Reply:
done